modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -3,87 +3,91 @@
-
-. Configure SSH on the Windows VM by running the following Powershell script:
+. Create a new virtual machine in the vSphere client from the Windows Server 2004 ISO image. See link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.hostclient.doc/GUID-7834894B-DD17-4D59-A9BF-A33D02478521.html[VMware's documentation] for more information.
 +
-[source,posh]
-----
-# Powershell script to configure SSH on vSphere Windows VMs
-Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
-$firewallRuleName = "ContainerLogsPort"
-$containerLogsPort = "10250"
-New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
-Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
-Install-Module -Force OpenSSHUtils
-Set-Service -Name ssh-agent -StartupType 'Automatic'
-Set-Service -Name sshd -StartupType 'Automatic'
-Start-Service ssh-agent
-Start-Service sshd
-$pubKeyConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PubkeyAuthentication yes','PubkeyAuthentication yes'
-$pubKeyConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication yes'
-$passwordConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$authFileConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys','#AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys'
-$authFileConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$pubKeyLocationConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'Match Group administrators','#Match Group administrators'
-$pubKeyLocationConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-Restart-Service sshd
-New-item -Path $env:USERPROFILE -Name .ssh -ItemType Directory -force
-----
+[IMPORTANT]
+====
+The Windows Server 2004 VM must include the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. This patch is not available for the `Windows Server 1809` VM image.
+====
 
 . Install and configure VMware Tools version 11.0.6 or greater on the Windows VM. See the link:https://docs.vmware.com/en/VMware-Tools/index.html[VMware Tools documentation] for more information.
 
 . After installing VMware Tools on the Windows VM, verify the following:
+
 .. The `C:\ProgramData\VMware\VMware Tools\tools.conf` file has the following entry:
```

Comment:
@jrvaldes please look into this.

Reply:
@sgaoshang is right, no tools.conf file can be found, since there are missing steps in the VMware Tools configuration. See [Setup VMware Tools](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-golden-image.md#setup-vmware-tools).

>In case the configuration file does not exist, VMware Tools installs an example configuration file in the same directory as the location above, or you can download it from the open-vm-tools repository.

Reply:
@jrvaldes Your statement `For example in Windows Server version 2004 the VMware Tools configuration file can be found at` is a bit misleading then. It made me think that such a file would exist which the user can then modify. Are you saying that `C:\ProgramData\VMware\VMware Tools\` is the location the file should be present? And if it is not, then the user should create it?

Reply:
Following  [VMware documentation](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-EA16729B-43C9-4DF9-B780-9B358E71B4AB.html) "If the configuration file does not exist, create it", which is covered in the next statement `In case the configuration file does not exist, VMware Tools installs an example configuration file in the same 
directory as the location above, or you can download it...`

Reply:
> Are you saying that C:\ProgramData\VMware\VMware Tools\ is the location the file should be present?

Yes


> And if it is not, then the user should create it?

Yes

Reply:
OK, this needs to be clarified in [Setup VMware Tools](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-golden-image.md#setup-vmware-tools). Having statements starting with `For example` is a bit confusing.